### PR TITLE
[Release] Remove HdInfoBox, Add HdAlert, HdButton, HdLink, extend HdInput behavior, update project metadata, update dependencies

### PR DESCRIPTION
**Pull request by SinisaG**
#### Breaking change
- Remove HdInfoBox #178 

#### Features 
- Add HdAlert #178
- Add HdButton #158 
- Add HdLink #176 
- Emit 'blur' and 'focus' from HdInput #200
- Watch type on HdInput #202
 
#### Tasks
- Add license and enhance metadata in package.json #188 
- Update demo branch #186 
- Upgrade dependencies #170 

Due to #186 we need to update cloudfront configuration once this is released. So I'm opening a PR already - to leave this note. ;)

### CODE REVIEW

**Comment by viniciuskneves** [comment]
@SinisaG please add the `HdLink` as well.

**Comment by SinisaG** [comment]
@viniciuskneves : Will do, no worries. I'll wait a bit more and release new version early next week - maybe some other PRs still make it in. 

Strange we have a conflict. :thinking:  I'll check what happened there also, when releasing.

File: AUTHORS [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# of contributors, see the revision history in source control.
+# List is order alphabetically based on last name
+
+Ilyas Amezouar <ilyas.amezouar@homeday.de>
+Juan Denis <juan.denis@homeday.de>
+Sinisa Grubor <sinisa.grubor@homeday.de>
+Oleg Ivaniv
+Vinicius Kiatkoski Neves <vinicius.neves@homeday.de>
+Daniele Rapisarda <daniele.rapisarda@homeday.de>
+Ante Scepic <ante.sepic@homeday.de>
```
Comment (OriginalEXE) [inline AUTHORS]:
```suggestion
Ante Sepic <ante.sepic@homeday.de>
```

A small typo :)

**Comment by ilyasmez** [comment]
@SinisaG is there anything that is stopping us from merging this PR and bumping the major?
The only PR that is still open is for a new component that can be added later in a minor release.

And @homeday-de/frontend-engineering (especially me) maybe we should start squash merging our PRs to develop in the hope of having a cleaner commit history 😄

File: AUTHORS (hunk 1 above)
Comment (SinisaG) [reply AUTHORS]: Oh, sorry about that. :see_no_evil: It happens often when I type. Fixed in #201 .

**Comment by SinisaG** [comment]
@ilyasmez : Nothing stopping us. I've updated the change log and prepared the release. Will merge tomorrow #201 and release. :rocket:

**Pull request merged**